site/docs/v1/tech/apis/_tenant-request-body.adoc

```diff
@@ -164,6 +164,22 @@ include::_tenant-generator-type-config-rules.adoc[]
 [field]#tenant.externalIdentifierConfiguration.emailVerificationIdTimeToLiveInSeconds# [type]#[Integer]# [required]#Required# [since]#Available since 1.8.0#::
 The time in seconds until a email verification Id is no longer valid and cannot be used by the Verify Email API. Value must be greater than 0.
 
+[field]#tenant.externalIdentifierConfiguration.emailVerificationOneTimeCodeGenerator.length# [type]#[Integer]# [required]#Required# [since]#Available since 1.27.0#::
++
+:generator_type: emailVerificationIdGenerator
+:generator_display_name: email verification one time code
+include::_tenant-generator-length-config-rules.adoc[]
+:generator_type!:
+:generator_display_name!:
++
+
+[field]#tenant.externalIdentifierConfiguration.emailVerificationOneTimeCodeGenerator.type# [type]#[String]# [required]#Required# [since]#Available since 1.27.0#::
++
+:generator_display_name: email verification one time code
+include::_tenant-generator-type-config-rules.adoc[]
+:generator_display_name!:
++
+
```

Comment:
These new values are not required, they should be optional, and the default value should be documented. 

https://github.com/FusionAuth/fusionauth-api/blob/22edb434e2fffabd68e78036c4107e454c2adc6d/src/main/java/io/fusionauth/api/service/system/DefaultTenantService.java#L454

https://github.com/FusionAuth/fusionauth-api/blob/22edb434e2fffabd68e78036c4107e454c2adc6d/src/main/java/io/fusionauth/api/service/system/DefaultTenantService.java#L590